src/lib/actions-watcher.js

```diff
@@ -91,6 +91,8 @@ function createChangeHandler (watcherOptions) {
     try {
       aioLogger.debug(`${filePath} has changed. Redeploying actions.`)
       const filterActions = getActionNameFromPath(filePath, watcherOptions)
+      // this is happening, but its not showing up because verbose is usually off
+      // this might be more important and worthy of signalling to the user
```

Comment:
I think this is important if we can define the importance of the "non-action file" is (more files are covered by your new changes) - the user might not know why their changes are not showing up otherwise